firstcup-war/pom.xml

```diff
@@ -17,7 +17,7 @@
     <parent>
         <groupId>jakarta.firstcup</groupId>
         <artifactId>examples</artifactId>
-        <version>8.1-SNAPSHOT</version>
+        <version>9.1-SNAPSHOT</version>
```

Comment:
The `project.version` should be 9.0-SNAPSHOT and not 9.1-SNAPSHOT

Reply:
Fixed, thank you